app/templates/me/index.hbs

```diff
@@ -31,8 +31,10 @@
 
     <p>
         If you want to use package commands from the command line, you will need to
-        login with <code>cargo login</code> using one of the tokens listed below. <br/>
-        When working in shared environments, supplying the token on the command line could expose it to other users, so it is more secure to supply a token when prompted. 
+        login with <code>cargo login (token)</code> using one of the tokens listed below. <br/>
+        When working in shared environments, supplying the token on the command line could 
+        expose it to prying eyes via command line history. To avoid this, enter <code>cargo login</code>
```

Comment:
I'm worried that if we put the `ps -a` notes in there, we'll have to explain the full background context, and that starts adding noise to what is an instruction for shared environments. We're avoiding the problem anyway if we tell people to use `cargo login` without the token insertion. Thoughts?